src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -8019,12 +8046,15 @@ namespace SurfaceGeometry {
 					kivaManager.settings.deepGroundBoundary = HeatBalanceKivaManager::KivaManager::Settings::ZERO_FLUX;
 				} else if (SameString(cAlphaArgs( alpF ), "GroundWater")) {
 					kivaManager.settings.deepGroundBoundary = HeatBalanceKivaManager::KivaManager::Settings::GROUNDWATER;
-				} else /* if (SameString(cAlphaArgs( alpF ), "Autoselect")) */ {
+				} else if (SameString(cAlphaArgs( alpF ), "Autoselect")) {
```

Comment:
Autoselect is fine, I was just more curious. But that is a good point about needing to transition it now.